collections-lists/CloneLists/CloneListsTests/Tests.cs

```diff
@@ -0,0 +1,63 @@
+using CloneLists;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CloneListsTests
+{
+    [TestClass]
+    public class Tests
+    {
+        [TestMethod]
+        public void verifylistwascloned_usingToListExample()
+        {
+            var carList = new List<string>() { "Porsche", "Corvette", "Bugati" };
+            ToListExample tle = new ToListExample();
+            List<string> clonedCarList = tle.RunToListExample(carList);
+
+            Assert.AreEqual(carList[0], clonedCarList[0]);
+            Assert.AreEqual(carList[1], clonedCarList[1]);
+            Assert.AreEqual(carList[2], clonedCarList[2]);
```

Comment:
You can assert the collections with CollectionAssert.AreEqual since you're using Microsoft Testing Framework. No need to check for each property separately.